src/examples/schedule-run-fetching-tests.yml

```diff
@@ -0,0 +1,35 @@
+description: >
+  Schedule functional tests via automated run on Testlio platform.
+  In this example we are assuming that the tests are not part of the same codebase
+  and we need to get them from somewhere.
+usage:
+  version: 2.1
+  orbs:
+    testlio: testlio/testlio@1.0.1
+  workflows:
+    schedule-run-for-a-project:
+      jobs:
+        - fetch-functional-tests
+        - testlio/schedule-run:
+            requires:
+              - fetch-functional-tests
+            attach-workspace: true
+            token: "TESTLIO_RUN_API_TOKEN"
+            test-config: "./functional-tests-folder/test-config.json"
+            project-config: "./functional-tests-folder/project-config.json"
+
+  fetch-functional-tests:
```

Comment:
This belongs under a `jobs` key.

Reply:
This job also has no executor,

example:
```
docker:
  - image: cimg/node:lts
```

Reply:
Good catch @KyleTryon, thanks!

I have fixed it. PTAL